doc/userdoc/installation/cmake_options.rst

```diff
@@ -28,70 +28,116 @@ Options for configuring NEST
+
+**Change compilation behavior:**
+
++---------------------------------------------+----------------------------------------------------------------+
+| -Dstatic-libraries=[OFF|ON]                 | Build static executable and libraries. [default=OFF]           |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-optimize=[OFF|ON|<list;of;flags>]    | Enable user defined optimizations. Separate multiple flags by  |
+|                                             | ';'. [default OFF, when ON, defaults to '-O3']                 |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-warning=[OFF|ON|<list;of;flags>]     | Enable user defined warnings. Separate  multiple flags by ';'. |
+|                                             | [default ON, when ON, defaults to '-Wall']                     |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-debug=[OFF|ON|<list;of;flags>]       | Enable user defined debug flags. Separate  multiple flags by   |
+|                                             | ';'. [default OFF, when ON, defaults to '-g']                  |
++---------------------------------------------+----------------------------------------------------------------+
+|-Dwith-intel-compiler-flags=[<list;of;flags>]| User defined flags for the Intel compiler. Separate multiple   |
+|                                             | flags by ';'. [defaults to '-fp-model strict']                 |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-libraries=[<list;of;libraries>]      | Link additional libraries. Give full path. Separate multiple   |
+|                                             | libraries by ';'. [default OFF]                                |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-includes=[<list;of;includes>]        | Add additional include paths. Give full path without '-I'.     |
+|                                             | Separate multiple include paths by ';'. [default OFF]          |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-defines=[<list;of;defines>]          | Additional defines, e.g. '-DXYZ=1'. Separate multiple defines  |
+|                                             | by ';'. [default OFF]                                          |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-version-suffix=[string]              | Set a user defined version suffix. [default '']                |
```

Comment:
What does this one actually do? Where is the suffix applied?

Reply:
@heplesser the suffix is added to the end of the NEST version. I tried building NEST with the suffix "test" on this branch, when calling `nest.__verision__` I then got 
```
>> nest.__version__
   'git_cmake_options-test@12ad267'
```